Detectors/MUON/MID/Base/include/MIDBase/Constants.h

```diff
@@ -77,14 +77,16 @@ class Constants
   /// @param rpc RPC ID (0-8)
   inline static int getDEId(bool isRight, int chamber, int rpc)
   {
-    int deOffset = (isRight) ? 0 : 36;
-    return deOffset + 9 * chamber + rpc;
+    int deOffset = (isRight) ? 0 : mNDetectionElementsPerSide;
+    return deOffset + mNRPCLines * chamber + rpc;
   }
 
-  static constexpr int mNChambers = 4;           ///< Number of chambers
-  static constexpr int mNDetectionElements = 72; ///< Number of RPCs
-  static constexpr int mNLocalBoards = 234;      ///< Number of local boards per chamber
-  static constexpr int mNStripsBP = 16;          ///< Number of strips in the Bending Plane
+  static constexpr int mNChambers = 4;                  ///< Number of chambers
+  static constexpr int mNDetectionElements = 72;        ///< Number of RPCs
+  static constexpr int mNLocalBoards = 234;             ///< Number of local boards per chamber
+  static constexpr int mNStripsBP = 16;                 ///< Number of strips in the Bending Plane
+  static constexpr int mNDetectionElementsPerSide = 36; ///< Number of detection elements per side
+  static constexpr int mNRPCLines = 9;                  ///< Number of RPC lines
```

Comment:
This is a nice PR. Just one more tiny comment: In principle our coding rules  state that static constexpr numbers should not carry the notion of a member variable. (In principle they should just be `NChambers` etc if I understand correctly or since it is static `sNChambers`).

Reply:
Hi Sandro,
you're right. Sorry, I overlooked the naming conventions for static data members. Should be fixed now.
Thanks a lot for the careful and instructive review!
Cheers,
Diego